signalfx/resource_signalfx_aws_integration.go

```diff
@@ -248,6 +249,13 @@ func integrationAWSResource() *schema.Resource {
 				Default:     false,
 				Description: "Indicates that Splunk Observability should only sync recommended statistics",
 			},
+			"metric_streams_managed_externally": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  false,
+				Description: "If set to true, Splunk Observability Cloud accepts data from Metric Streams managed from the AWS console. " +
+					"The AWS account sending the Metric Streams and the AWS account in the Splunk Observability Cloud integration have to match.",
```

Comment:
Maybe it is worth to add here and in the markdown that this field is effective only when `use_metric_streams_sync = true`?